src/legacy/core_plugins/kibana/public/management/sections/objects/lib/resolve_saved_objects.ts

```diff
@@ -81,7 +99,7 @@ async function importIndexPattern(doc, indexPatterns, overwriteAll, confirmModal
     // We can override and we want to prompt for confirmation
     const isConfirmed = await confirmModalPromise(
       i18n.translate('kbn.management.indexPattern.confirmOverwriteLabel', {
-        values: { title: this.title },
+        values: { title },
```

Comment:
Seems like an error after a probable extraction from a react component. I don't even know how the `this` reference was working (probably it wasn't).